mail/views.py

```diff
@@ -31,6 +32,41 @@
 log = logging.getLogger(__name__)
 
 
+class LearnerMailView(GenericAPIView):
+    """
+    View class that handles HTTP requests to learner mail API
+    """
+    serializer_class = GenericMailSerializer
+    authentication_classes = (
+        authentication.SessionAuthentication,
+        authentication.TokenAuthentication,
+    )
+    permission_classes = (permissions.IsAuthenticated, UserCanMessageLearnersPermission, )
```

Comment:
We need to also verify that the staff is staff of a program that the user is enrolled in. We don't have an extra permission in the search view because we add a filter on the programs the staff is enrolled in via Elasticsearch, but here we should check this in a permission class